contracts/interfaces/IWavsServiceHandler.sol

```diff
@@ -2,9 +2,19 @@
 pragma solidity ^0.8.0;
 
 interface IWavsServiceHandler {
+    struct SignatureData {
```

Comment:
Can we document these things? : )

What is a `referenceBlock` for example?

Reply:
does [this](https://github.com/BreadchainCoop/eigenlayer-middleware/blob/848032a7e22b2e6a2ab7835a45e9b39bb16d41a5/src/unaudited/ECDSAStakeRegistry.sol#L417) help explain it?